.github/workflows/release-on-merge.yml

```diff
@@ -0,0 +1,64 @@
+name: Trigger release on merge
+run-name: Starting release for ${{ github.actor }} PR merge
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - develop
+
+jobs:
+  on_pr_merge:
```

Comment:
I would rather name it what it does, then when it is executed. 

So I guess `trigger-ui-release` or `tag-ui-release` or something would fit better.

Reply:
`trigger-ui-release` make sense 👍